library/src/main/java/com/github/paolorotolo/appintro/AppIntroFragment.java

```diff
@@ -15,14 +16,35 @@
 
 public final class AppIntroFragment extends AppIntroBaseFragment {
     public static AppIntroFragment newInstance(CharSequence title, CharSequence description, int imageDrawable, int bgColor) {
-        return newInstance(title, description, imageDrawable, bgColor, 0, 0);
+        return newInstance(title,"" ,description,"", imageDrawable, bgColor, 0, 0);
```

Comment:
No. It needs "" or `null` to ignore the requirement of custom typeface & use the default OS typeface